### PR TITLE
JS-464 Update ESLint plugin metadata and fix scripts

### PR DESCRIPTION
**Pull request by saberduck**
[JS-464](https://sonarsource.atlassian.net/browse/JS-464)

[JS-464]: https://sonarsource.atlassian.net/browse/JS-464?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/jsts/src/rules/S6299/meta.ts [hunk 1]
```diff
@@ -20,10 +20,11 @@ export const meta = {
   type: 'problem',
   docs: {
     description: 'Disabling Vue.js built-in escaping is security-sensitive',
-    recommended: true,
+    recommended: false,
```
Comment (vdiez) [inline packages/jsts/src/rules/S6299/meta.ts:23]: why this change? this comes from RSPEC, cannot be edited directly.

File: package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "build:fast": "npm run bridge:build:fast && npm run _:plugin:pre-build && npm run plugin:build:fast",
     "bf": "npm run build:fast",
     "new-rule": "tsx tools/newRule.ts",
-    "generate-meta": "tsx tools/generate-meta.ts && cd packages/jsts/src/rules && npm run eslint-docs",
+    "generate-meta": "tsx tools/generate-meta.ts && npm run format && cd packages/jsts/src/rules && npm run eslint-docs",
```
Comment (vdiez) [inline package.json:13]: this is fixed in the new PR

File: tools/generate-meta.ts [hunk 3]
```diff
@@ -86,21 +86,16 @@ function generateMetaForRule(ruleDir: string, ruleId: string) {
...
- * License as published by the Free Software Foundation; either
- * version 3 of the License, or (at your option) any later version.
+ * modify it under the terms of the Sonar Source-Available License Version 1, as published by SonarSource SA.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the Sonar Source-Available License for more details.
```
Comment (vdiez) [inline tools/generate-meta.ts:94]: this is fixed in the new PR

**Review by vdiez — CHANGES_REQUESTED** [review]
please limit changes to new number of rules in README and the fix on count-rules

File: packages/jsts/src/rules/S6299/meta.ts (hunk 1 above)
Comment (saberduck) [reply packages/jsts/src/rules/S6299/meta.ts:23]: it's coming from rspec

Comment (vdiez) [reply packages/jsts/src/rules/S6299/meta.ts:23]: ok, didn't see that. coming from https://github.com/SonarSource/rspec/pull/4541

in any case, once https://github.com/SonarSource/SonarJS/pull/4917 is merged, this files are now autogenerated on `npm ci` and added go git ignore

**Review by vdiez — APPROVED** [review]
